tests/fuzz/fuzz_common.h

```diff
@@ -0,0 +1,62 @@
+//
+// Created by profanter on 18.07.17.
+// Copyright (c) 2017 fortiss GmbH. All rights reserved.
+//
+
+#ifndef OPEN62541_FUZZ_COMMON_H_H
```

Comment:
I see.
Is it possible to build the fuzzing targets also with cmake?
Similar to how the examples are built.
That way, we have more freedom to compile several source files into one fuzzing target.